.github/workflows/test.yml

```diff
@@ -11,9 +11,9 @@ jobs:
   test:
     strategy:
       matrix:
-        node-version: [16, 18]
+        node-version: [16, 18, 20]
     runs-on: ubuntu-latest
-    name: Test Node.js ${{ matrix.node-version }}
+    name: Test Node.js ${{ matrix.node-version }} with prom-client ${{ matrix.prom-client-version }}
```

Comment:
No need to add prom client there. This is CI name. Let it be just `Test Node.js ${{ matrix.node-version }}`